pkg/kubectl/stop.go

```diff
@@ -337,6 +333,8 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	}
 	if timeout == 0 {
 		numReplicas := ss.Spec.Replicas
+
+		// BUG: this timeout is never used.
```

Comment:
The original author of this function and unused value issue (in [30f3cb9](https://github.com/kubernetes/kubernetes/commit/30f3cb9d26fe368388502a362f9577bf20fa5d65#diff-2076b6236060a0cca0937ac2907ab263R339)) is github.com/bprashanth but he has since moved to another project.

I will make a new PR for discussion of this particular bug. For now, the comment will warn people that there might be an issue.